dapps/evancore.vue.libs/src/components/dapp-wrapper/dapp-wrapper.vue

```diff
@@ -26,6 +26,9 @@
     <evan-logout ref="evanLogout" :disableButton="true"></evan-logout>
 
     <div class="dapp-wrapper-body"
+      :class="{
+        'remove-header': onboarding || login,
```

Comment:
You could remove padding without adding a new class:

```suggestion
        'pt-0': onboarding || login,
```